### ISSUE TITLE
Fixed gutters issue

### ISSUE THREAD

**Issue by dekisha** [at-import/Singularity#243]
When gutter is set as fixed, gutters are added as paddings instead of margins.

So when you set your last column as last with @include float-span(4, last); for example last column loses its gutter but occupies whole available space.

I described this issue in detail on stackoverflow:

https://stackoverflow.com/questions/46179004/singularity-grid-fixed-gutters-issue

**Comment by cameronprince** [comment]
@dekisha, did you ever find a solution to this problem? Fixed gutters isn't really viable as it is now because the sizing is inconsistent.

**Comment by dekisha** [comment]
@cameronbprince no, I haven't find a solution to my issue. I understand that sizing is inconsistent, but design requirements were to always have fixed gutter of 20px. IMO there shouldn't be an option to use fixed gutters in singularity if it's impossible to align left and right edge with container. 

Nevertheless I am using flexbox for layout now, so this is issue is no longer that much relevant to me. But I would really want to know if this is a bug or I am doing something wrong?

Stackoverflow question hasn't been answered as well as you can notice.

**Comment by scottkellum** [comment]
Thanks for the bug report and yes it sounds like it’s either a bug with the code or the documentation.

However Singularity is no longer being maintained and we recommend using CSS grid or flexbox instead as they do many of the same things more simply and efficiently.
